### PR TITLE
Related #1070: Fix Codacy issues for files in org.loklak.api.search

### PR DESCRIPTION
**Pull request by singhpratyush**
Related #1070.

* EventBriteCrawlerService.java
  - Use one line for each declaration
  - https://www.codacy.com/app/sudheesh1995/loklak_server/file/6497733425/issues/source?bid=3495500&fileBranchId=3495500

* GenericScraper.java
  - Indentation fix
  - New line before EOF

* GithubProfileScraper.java
  - Remove trailing whitespaces

* MeetupsCrawlerService.java
  - Use one line for each declaration
  - https://www.codacy.com/app/sudheesh1995/loklak_server/file/6497733676/issues/source?bid=3495500&fileBranchId=3495500

* SearchServlet.java
  - Indentation fix

* SuggestServlet.java
  - Position literals first in String comparisons
  - Fields should be declared at the top of the class
  - https://www.codacy.com/app/sudheesh1995/loklak_server/file/6497733640/issues/source?bid=3495500&fileBranchId=3495500

* WeiboUserInfo.java
  - Switch statements should have a default label
  - https://www.codacy.com/app/sudheesh1995/loklak_server/file/6497733550/issues/source?bid=3495500&fileBranchId=3495500

### Short description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by mariobehling** [comment]
@vibhcool Please help to review.

File: src/org/loklak/api/search/MeetupsCrawlerService.java [hunk 1]
```diff
@@ -63,7 +63,7 @@ public JSONObject serviceImpl(Query call, HttpServletResponse response, Authoriz
 	}
 
 	public static SusiThought crawlMeetups(String url) {
-
```
Comment (vibhcool) [inline src/org/loklak/api/search/MeetupsCrawlerService.java]: There is a lot of spacing replacements with new line. May be this is out of bound of the PR

Comment (singhpratyush) [inline src/org/loklak/api/search/MeetupsCrawlerService.java]: The spacing is not replaced with a newline, I have just removed the unnecessary trailing spaces.

File: src/org/loklak/api/search/GenericScraper.java [hunk 2]
```diff
@@ -6,12 +6,12 @@
  *  modify it under the terms of the GNU Lesser General Public
  *  License as published by the Free Software Foundation; either
  *  version 2.1 of the License, or (at your option) any later version.
- *  
+ *
  *  This library is distributed in the hope that it will be useful,
  *  but WITHOUT ANY WARRANTY; without even the implied warranty of
  *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
  *  Lesser General Public License for more details.
- *
```
Comment (vibhcool) [inline src/org/loklak/api/search/GenericScraper.java]: spacing replacements with new line!!! need to be fixed

Comment (singhpratyush) [inline src/org/loklak/api/search/GenericScraper.java]: The spacing is not replaced with newlines. It is just trailing whitespace which is removed.

**Pull request merged**